modules/installation-and-upgrade/pages/hardware-requirements.adoc

```diff
@@ -216,6 +217,15 @@ As a result of this behavior:
 * The [path]``/var/lib/containers/storage/volumes/uyuni-proxy-squid-cache/`` directory can be small on the {productname} Server compared to the proxy.
   For a guide to size estimation, see the <<server-hardware-requirements>> section.
 
+[NOTE]
+====
+In general, {suse} recommends to adjust the value for the cache directory to about 60 % of available free space.
+Users can set the [option]``cache_dir`` option in the [path]``squid.conf`` manually.
+
+For more information, see xref:specialized-guides:large-deployments/tuning.adoc#tune_proxy[].
```

Comment:
This is not correct.

1) 60% is very small here because there is nothing else taking the storage.

To be safe we can follow rule on squid configuration, that is using 80% of the space.

We recommend to be this 100G, same as the storage for that partition. Which will be the default in 5.1.

2) users cannot modify `squid.conf` anymore. This value is set when generating proxy configuration on the server.